driver/lib/commands/execute.ts

```diff
@@ -242,3 +245,11 @@ const dragAndDropWithCommandExtension = async (
   };
   return await self.socket!.executeSocketCommand(commandPayload);
 };
+
+async function getTextWithCommandExtension(self: FlutterDriver, params: { findBy: string; }) {
+    const payload = {
+        command: 'getTextWithCommandExtension',
+        findBy: params.findBy,
+    };
+    return await self.socket!.executeSocketCommand(payload);
+}
```

Comment:
Could you follow two indentations basis?

Reply:
yep, I fixed it.